src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec

```diff
@@ -57,5 +57,20 @@
     <file src="Dlls\MSBuildTask\Microsoft.CSharp.Core.targets" target="contentFiles\any\any" />
     <file src="Dlls\MSBuildTask\Microsoft.VisualBasic.Core.targets" target="contentFiles\any\any" />
     <file src="$thirdPartyNoticesPath$" target="" />
+
```

Comment:
(Doesn't apply to this line, but putting it here to start a thread.)

Double check that none of the satellites are being added to the .vsix yet. I don't see SatelliteDllsProjectOutputGroup anywhere in code so I suspect it's fine, but we need to make sure that we don't clobber the closed source satellites in VS with the ones that will be generated all English until the first hand-back that will recycle the translations.